lib/semian/net_http.rb

```diff
@@ -33,11 +33,12 @@ def semian_identifier
     end
 
     DEFAULT_ERRORS = [
-      ::Timeout::Error, # includes ::Net::ReadTimeout and ::Net::OpenTimeout
+      ::Timeout::Error, # includes ::Net::ReadTimeout
       ::SocketError,
       ::Net::HTTPBadResponse,
       ::Net::HTTPHeaderSyntaxError,
       ::Net::ProtocolError,
+      ::Net::OpenTimeout, # https://github.com/ruby/ruby/blob/trunk/lib/net/http.rb#L902
```

Comment:
Says [here](https://github.com/ruby/ruby/blob/trunk/lib/net/protocol.rb#L72) it does indeed inherit from `::Timeout::Error`